src/app/pages/Login.js

```diff
@@ -19,16 +22,14 @@ function Login() {
         })
     };
 
-    const { currentUser } = useContext(AuthContext)
-
-    return (
-        !!currentUser
-        ? <Navigate to='/' />
-        : (
-            <div className="flex justify-center items-center bg-green-700 h-[100vh]">
-                <GoogleButton onClick={signInWithGoogle} />
-            </div>
-        )
+    return loading ? (
+        <h1>Loading...</h1>
```

Comment:
If the user is undefined, the page is loading. Otherwise, we can handle rendering as we previously were.